web/packages/teleport/src/Discover/Kubernetes/EnrollEKSCluster/ManualHelmDialog.tsx

```diff
@@ -89,3 +163,17 @@ const StyledBox = styled(Box)`
   padding: ${props => `${props.theme.space[3]}px`};
   border-radius: ${props => `${props.theme.space[2]}px`};
 `;
+
+const Spin = styled(Box)`
```

Comment:
what's different with this spinner than the spinner we already have? 🤔

Reply:
bump, we have an [Indicator](https://github.com/gravitational/teleport/blob/5302e5ed4f5b089563b99ce91da9b77ff75243d2/web/packages/design/src/Indicator/Indicator.jsx) component?

Reply:
Sorry, missed this one. I just didn't find spinner component 😅  I searched for "spinner" and found only an icon and then some different css implementations sprinkled around the code. Thanks for pointing out the indicator component!